test/e2e/framework/addon/globals.go

```diff
@@ -142,18 +142,18 @@ func ProvisionGlobals(ctx context.Context, cfg *config.Config) error {
 }
 
 type loggableAddon interface {
-	Logs() (map[string]string, error)
+	Logs(ctx context.Context) (map[string]string, error)
```

Comment:
Unrelated change: this interface was no longer matching the definitions in https://github.com/cert-manager/cert-manager/blob/71f43d5fa0b619d8b80ce4a168d75e95bab9486a/test/e2e/framework/addon/chart/addon.go#L226 and https://github.com/cert-manager/cert-manager/blob/71f43d5fa0b619d8b80ce4a168d75e95bab9486a/test/e2e/framework/addon/vault/vault.go#L440.